virtool/fake/factory.py

```diff
@@ -59,14 +60,14 @@ class WorkflowTestCase:
 class TestCaseDataFactory:
     """Initialize the database with fake data for a test case."""
 
-    def __init__(self, app: App, user_id: str, job_id: str = None):
+    def __init__(self, settings: Settings, app: App, user_id: str, job_id: str = None):
```

Comment:
I don't think you need to pass in `settings` separately here.

Can't you just use `get_data_from_app` on the `app` object that's also a parameter here?

Reply:
Same realisation as in fixtures/client.py